webapp/templates/team/partials/clarification_list.html.twig

```diff
@@ -64,7 +41,15 @@
 
                 <td>
                     <a data-ajax-modal data-ajax-modal-after="markSeen" href="{{ link }}">
-                        {{ clarification.summary | markdown_to_html | sanitize_html('app.clarification_sanitizer') }}
+                        {%- if clarification.sender is null and clarification.recipient is not null %}
+                            <i class="fa-solid fa-reply" title="Reply to you"></i>&nbsp;
+                        {%- endif %}
+                        <i>{{ clarification.summary
+                            | markdown_to_html
+                            | replace({'<p>': '', '</p>': ''})
+                            | sanitize_html('app.clarification_sanitizer')
+                            | raw
```

Comment:
Why do we make this raw?

Reply:
because the last filter already sanitizes it, so no double sanitation required